src/Moryx.Communication.Mqtt.Endpoints/ResourceRpcEndpoint.cs

```diff
@@ -0,0 +1,99 @@
+// Copyright (c) 2025, Phoenix Contact GmbH & Co. KG
+// Licensed under the Apache License, Version 2.0
+
+using Microsoft.Extensions.Logging;
+using Moryx.AbstractionLayer.Identity;
+using Moryx.AbstractionLayer.Resources;
+using Moryx.Communication.Mqtt.Builders;
+using Moryx.Communication.Mqtt.Components;
+using Moryx.Communication.Mqtt.Converters;
+using Moryx.Serialization;
+using MQTTnet;
+
+namespace Moryx.Communication.Mqtt.Endpoints;
```

Comment:
Open Question: Should we rather place all the endpoint in this namespace (at least that is how I understand your intend) or should we rather put the `ResourceRpcEndpoint` into the `Moryx.AbstractionLayer.Resources.Endpoints` package? 🤔

Reply:
Its a special case.. this Endpoint has a dependency to mqtt stuff, and I definitely don't want a reference to that in `Moryx.AbstractionLayer.Resources.Endpoints`. For my POV, this endpoint should be separated because I also dont see a reference of `Moryx.AbstractionLayer` in `Moryx.Communication.Mqtt`